packages/mds-types/transformers/0_4_1_to_1_0_0/vehicle_events.ts

```diff
@@ -98,7 +99,7 @@ export function convert_v0_4_1_vehicle_event_to_v1_0_0(event: VehicleEvent_v0_4_
   } = event
 
   const { event_type: new_event_type, vehicle_state } = map_v0_4_1_vehicle_event_fields_to_v1_0_0_fields(
-    event_type,
+    event_type as INGESTABLE_VEHICLE_EVENT,
```

Comment:
What if before this line we check to see if `event_type === 'register'`, and throw a named exception for it? I'm a little worried that this could cause a hard-to-trace runtime exception if we don't do that.

Reply:
Feels kinda overly defensive because it's not an event type we expect to ever have to transform, but sure.

Reply:
Is this typecast still needed now?